app/services/foreman_salt/fact_parser.rb

```diff
@@ -0,0 +1,78 @@
+module ForemanSalt
+  class FactParser < ::PuppetFactParser
```

Comment:
does it really make sense to inherit from puppet? is there anything in the puppet parser that can be moved to the parent base class instead?